packages/base/any/kernels/5.10-lts/patches/0025-net-Added-MII-support-to-mvpp2-driver.patch

```diff
@@ -0,0 +1,132 @@
+From 67cbf15c026cec3887e19d02bd82164ecfdd697a Mon Sep 17 00:00:00 2001
+From: Elad Nachman <enachman@marvell.com>
+Date: Wed, 28 Jul 2021 13:34:39 +0300
+Subject: [PATCH 2/4] net: Added MII support to mvpp2 driver
+
+This patch adds MII support to the Marvell PPv2
+(Packet Processor v2) 1/10 Gbps adapter driver.
```

Comment:
1.  Please re-flow for 75 characters per line.
2.  Please mention the datasheet name and revision.